### PR TITLE
Add minter address

### PR DESCRIPTION
**Pull request by AStox**
Renames the `registry` address to `minter` and adds both a `registry` and `minter` address to the `mintIssuePriceLock()` function.

### CODE REVIEW

**Review by xmxanuel — COMMENTED** [review]
Looks good but you still need to adjust the tests for `mintIssuePriceLock`

**Review by xmxanuel — DISMISSED** [review]

File: cache/solidity-files-cache.json [hunk 1]
```diff
@@ -0,0 +1,1548 @@
+{
```
Comment (xmxanuel) [inline cache/solidity-files-cache.json]: I think this file can be removed from the commit

**Review by xmxanuel — CHANGES_REQUESTED** [review]

File: cache/solidity-files-cache.json (hunk 1 above)
Comment (AStox) [reply cache/solidity-files-cache.json]: huh... wonder why I have that

Comment (AStox) [reply cache/solidity-files-cache.json]: Removed and updated .gitignore

**Review by xmxanuel — APPROVED** [review]

**Review by hieronx — APPROVED** [review]

**Pull request merged**